daffodil-test-ibm1/src/test/resources/test-suite/ibm-contributed/dpaext2.tdml

```diff
@@ -327,7 +327,7 @@
 		<document><documentPart type="byte">000000000000003C</documentPart></document>
 		<infoset>
 			<dfdlInfoset xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-			  <testBinaryMilliseconds xsi:type="xs:dateTime">2009-12-24T05:15:00.060000+05:00</testBinaryMilliseconds>
+			  <testBinaryMilliseconds xsi:type="xs:dateTime">2009-12-24T05:15:00.060000+00:00</testBinaryMilliseconds>
```

Comment:
Interesting. The original test had a binaryCalendarEpoch in the +5 timezone, and so the expected infoset value was in the +5 timezone. I would expect that the output should be in the timezone of the dfdl:calendarTimeZone property. This is because with binaryCalendarSec/MilSec no timezone is in the data, and the dfdl:calendarTimeZone property says: This property provides the time zone that will be assumed if no time zone explicitly occurs in the data. Seems like the test was wrong, and even if dfdl:binaryCalendarEpoch includes a timezone, it shouldn't affect the infoset timezone. Thoughts?

Reply:
Ok, that explanation makes sense to me.

Reply:
I removed the time zones from the binaryCalendarEpoch because I think Steve noted before that it shouldn't be using the calendarPattern because that doesn't apply for binarySeconds and binaryMilliseconds. So when parsing the epochCalendar I am using the default pattern for dateTime which is "yyyy-MM-dd'T'HH:mm:ss". 

But if calendarTimeZone doesn't apply here either than this is wrong too since I was using it for epochCalendar. So without specifying a pattern for epochCalendar can we allow both "yyyy-MM-dd'T'HH:mm:ss" and "yyyy-MM-dd'T'HH:mm:ssZZZZ"? Then the first would use UTC.